hack/generate/samples/internal/go/memcached-with-webhooks/e2e_test_code.go

```diff
@@ -288,9 +288,9 @@ var _ = Describe("memcached", func() {
 
 			By("validating that pod(s) status.phase=Running")
 			getMemcachedPodStatus := func() error {
-				cmd = exec.Command("kubectl", "get",
-					"pods", "-l", "app=memcached", "-o", "jsonpath={.items[*].status}",
-					"-n", namespace,
```

Comment:
@NikhilSharmaWe we are very close :-) 

See that the e2e test is looking to check the node.status update which does not exist anymore : 
https://github.com/operator-framework/operator-sdk/blob/afc3a88a0ac7d694202c5ef0035553b2846d7ded/test/e2e/go/cluster_test.go#L181-L195

So, we can remove this one.

Then, add the check https://github.com/kubernetes-sigs/kubebuilder/blob/master/test/e2e/deployimage/plugin_cluster_test.go#L267-L279 after we validate that the pod is running.